api/model.go

```diff
@@ -1191,11 +1191,11 @@ func toSnapshotResource(s *longhorn.Snapshot) *Snapshot {
 			Id:   s.Name,
 			Type: "snapshot",
 		},
-		Snapshot: *s,
+		SnapshotInfo: *s,
```

Comment:
This will impact the output of API, so probably will impact UI? Have you seen any problems from UI?

Reply:
As confirmed by @PhanLe1010 , it will not impact the output.

note: the response of API actually depends on the schema-registered snapshot response structure as below w/ embedded structures, so decoding still works correctly.

https://github.com/longhorn/longhorn-manager/pull/1304/files#diff-22d9efdab1c8ceb9165fd0f1b5777cec7d012c74ec1d1f1d84c9e96d918b6fc0R74-R78